example/localStorage.html

```diff
@@ -0,0 +1,37 @@
+<body>
```

Comment:
Besides this example, is it worth mentioning in the README or the documentation?